quickget

```diff
@@ -1042,6 +1048,15 @@ function get_arcolinux() {
     echo "${URL}/${ISO} ${HASH}"
 }
 
+function get_biglinux() {
+    local HASH=""
+    local ISO=""
+    local URL="https://iso.biglinux.com.br/"
```

Comment:
Please could you remove the trailing "/" and use "${URL}/${ISO}" below as per the other 64 instances in `quickget` for consistency

Reply:
Hi @philclifford, thank you for your instructions. I wanted to let you know that I understood the instructions and have made the necessary changes. I apologize for the mistake I made earlier, and I appreciate your time and assistance. Thank you!